mistral/mistral.py

```diff
@@ -247,6 +247,12 @@ def sample(logits):
         type=float,
         default=1.0,
     )
+    parser.add_argument(
+        "--batch_size",
```

Comment:
I'm fine to make this an argument, but I think the name `batch_size` may mislead some people as it usually refers to batching over different examples not within one example.

Could you change it to something like `tokens_per_eval` or something a little more specific?

Reply:
Sounds good, changed to `tokens_per_eval`